src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -160,6 +166,16 @@
                       :disabled="true"
                     />
                   </v-col>
+
+                  <v-col cols="4" v-if="showPartnerRole" class="py-0">
+                    <v-checkbox
+                      class="mt-5"
+                      v-model="selectedRoles"
+                      :value="RoleTypes.PARTNER"
+                      :label="RoleTypes.PARTNER"
+                      :disabled="true"
```

Comment:
Always disabled, so readonly?

Reply:
Yes but the readonly style is incorrect.

Disabled:

![image](https://user-images.githubusercontent.com/10002889/162831790-99292e7b-96cc-4514-a96f-4b955fde60d0.png)

Readonly:

![image](https://user-images.githubusercontent.com/10002889/162831823-21c384b3-11eb-4995-b0ef-27427a1bda79.png)

Reply:
I guess, technically, disabled is more correct since sometimes the user can check one of these Roles boxes, just not in this scenario. Whereas readonly is more for presentation-only (IMO).

Reply:
Sounds good!